platform/ios/src/MGLMapView.mm

```diff
@@ -4401,17 +4413,12 @@ - (void)selectAnnotation:(id <MGLAnnotation>)annotation moveOnscreen:(BOOL)moveO
                 calloutPositioningRect = annotationView.frame;
                 [annotationView.superview bringSubviewToFront:annotationView];
 
-                [annotationView setSelected:YES animated:animateSelection];
+                [annotationView setSelected:YES animated:animated];
```

Comment:
This basically fixes #13055, but the `expose` parameter is now redundant to the `animated` parameter and should be removed.